commands/Administration/addcommand.js

```diff
@@ -23,7 +23,7 @@ class Addcommand extends Command {
 
     async run (message, args, data) {
         
-        let name = args[0];
+        let name = args[0].replace(/\n/g, "");;
```

Comment:
Generally, if a person adds a line break, it is to separate the command name from its content so I think it's better to do 
```Js
let name = args[0].split("\n")[0];
```
and
```Js
let answer = (args[0].split("\n")[1] || "") + args.slice(1).join(" ");
```

Reply:
perfect !